docs/permission/plugin-authors/02-adding-a-basic-permission-check.md

```diff
@@ -26,7 +26,7 @@ Let's navigate to the file `plugins/todo-list-common/src/permissions.ts` and add
   });
 ```
 
-For this tutorial, we've automatically exported all permissions from this file (see `plugins/todo-list-common/src/index.ts`). For the actual plugins that you author, you must export any permission authorized by the plugin so Backstage integrators can import them when writing policies.
+> Note: All permissions authorized by your plugin should be exported from a ["common-library" package](https://backstage.io/docs/local-dev/cli-build-system#package-roles). This allows Backstage integrators to reference them in frontend components and permission policies.
```

Comment:
I think it's worth keeping the fact that we've already done this work for them in this specific case (lest they confuse this note as an actionable step without a clear direction).

Reply:
Yeah good callout. How about we add that first sentence back in and keep this note below it?